frontend/src/containers/LandingPage.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '../styles/App.scss';
+import styles from '../styles/LandingPage.module.scss';
+import BlobLeft from '../assets/BlobLeft.svg';
+import BlobRight from '../assets/BlobRight.svg';
+import Button from '../components/Button';
+
+function LandingPage() {
+    return(
+        <div>
+            <img className={styles.blobleft} src={BlobLeft} alt="" />
+            <img className={styles.blobright} src={BlobRight} alt="" />
+            <div className={styles.header}>ACM QUIZTIME</div>
```

Comment:
Please make `ACM QUIZTIME` in one line and at the same position as the `ACM QUIZTIME` in the Universal Landing Page. You can reuse the SCSS classes from the Universal Landing Page here
<img width="1440" alt="Screen Shot 2020-09-02 at 2 03 10 PM" src="https://user-images.githubusercontent.com/15056430/92036422-0b74d380-ed25-11ea-8cde-9ec8eaf660b6.png">